packages/eslint-plugin/src/util/explicitReturnTypeUtils.ts

```diff
@@ -3,9 +3,14 @@ import {
   AST_NODE_TYPES,
   TSESLint,
 } from '@typescript-eslint/experimental-utils';
-import { isTypeAssertion, isConstructor, isSetter } from './astUtils';
-import { getFunctionHeadLoc } from './getFunctionHeadLoc';
-import { nullThrows, NullThrowsReasons } from './nullThrows';
+import {
+  isTypeAssertion,
+  isConstructor,
+  isSetter,
+  getFunctionHeadLoc,
+  nullThrows,
+  NullThrowsReasons,
+} from '.';
```

Comment:
we should avoid importing from `.` as it creates a cyclic dependency tree.
eg: `index` imports `explicitReturnTypeUtils` imports `index`.

Reply:
fixed